lib/winapi/winbase.h

```diff
@@ -11,6 +12,14 @@ extern "C"
 DWORD GetLastError (void);
 void SetLastError (DWORD error);
 
+void WINAPI OutputDebugStringA (LPCTSTR lpOutputString);
```

Comment:
[MS docs](https://docs.microsoft.com/en-us/windows/desktop/api/debugapi/nf-debugapi-outputdebugstringa) indicate that this should be located in a header called `debugapi.h`.

Reply:
I believe there's 2 different versions of this function: for userspace, and for kernel drivers.

https://source.winehq.org/WineAPI/OutputDebugStringA.html (and other languages of the MSDN article) claim it's in winbase.h, and I believe I found the location from a copy of an old DDK myself.

It's also possible that they moved the function over time to make winbase.h / windows.h more modular.